### PR TITLE
Awesome books es6 ver1

### PR DESCRIPTION
**Pull request by sajibarafatsiddiqui**
1) Configured Linters
2) Created a professional read me file
3) Maintained git flow.
4)Created two modules inside modules directory
5) Refractor the code following es6 rules

Thanks.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  contactsPage.classList.add('hide');
+
+  const showListOnly = () => {
+    formsPage.classList.add('hide');
+    contactsPage.classList.add('hide');
+    booksPage.classList.remove('hide');
+    booksTitle.classList.remove('hide');
+  };
+  booksLink.addEventListener('click', showListOnly);
+  function showContactOnly() {
```
Comment (prolajumokeoni) [inline index.js]: - [ ] Kindly use arrow syntax for functions, this is a requirement for this milestone 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @sajibarafatsiddiqui 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used 
✔️  Good PR title and Description



## Required Changes ♻️



- [x] Kindly make sure your app is working, currently, it is not possible to see the pages when I click on the nav links or add  & remove books from the list.
- see the video below: 

https://user-images.githubusercontent.com/69638013/212782684-22003a8b-9175-439d-b400-a4c96f5c1202.mp4



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


- Please always capitalize the first letter of your commit messages, this is a good practice to follow when writing good commit messages


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (sajibarafatsiddiqui) [reply index.js]: Oh yes I changed it to es6 . Thanks

Comment (sajibarafatsiddiqui) [reply index.js]: Ok The code is working in all browser now. Thanks

**Review by TadesseDev — APPROVED** [review]
### Hi @sajibarafatsiddiqui 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
ST3D


## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅
 - [ ]  Good commit history. ✅
 - [ ]  Good PR title/description. ✅
 - [ ]  All linters passed ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**